features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/impl/DatadogTracing.kt

```diff
@@ -5,42 +5,29 @@
  */
 package com.datadog.android.trace.impl
 
-import androidx.annotation.VisibleForTesting
 import com.datadog.android.Datadog
 import com.datadog.android.api.InternalLogger
 import com.datadog.android.api.SdkCore
 import com.datadog.android.api.feature.Feature
 import com.datadog.android.api.feature.FeatureSdkCore
-import com.datadog.android.internal.utils.tryCastTo
-import com.datadog.android.lint.InternalApi
 import com.datadog.android.trace.InternalCoreWriterProvider
-import com.datadog.android.trace.api.span.DatadogSpanLogger
-import com.datadog.android.trace.api.span.NoOpDatadogSpanLogger
 import com.datadog.android.trace.api.tracer.DatadogTracerBuilder
 import com.datadog.android.trace.api.tracer.NoOpDatadogTracerBuilder
-import com.datadog.android.trace.impl.internal.DatadogSpanLoggerAdapter
 import com.datadog.android.trace.impl.internal.DatadogSpanWriterWrapper
 import com.datadog.android.trace.impl.internal.DatadogTracerBuilderAdapter
-import com.datadog.android.trace.internal.data.NoOpCoreTracerWriter
+import com.datadog.android.trace.impl.internal.DatadogTracingInternalToolkit
+import com.datadog.android.trace.impl.internal.DatadogTracingInternalToolkit.ErrorMessages.DEFAULT_SERVICE_NAME_IS_MISSING_ERROR_MESSAGE
+import com.datadog.android.trace.impl.internal.DatadogTracingInternalToolkit.ErrorMessages.TRACING_NOT_ENABLED_ERROR_MESSAGE
+import com.datadog.android.trace.impl.internal.DatadogTracingInternalToolkit.ErrorMessages.WRITER_PROVIDER_INTERFACE_NOT_IMPLEMENTED_ERROR_MESSAGE
```

Comment:
do we want to share error messages? is it useful?

Reply:
as discussed, moved into `DatadogTracing`